### PR TITLE
Contrast issue with WISE blue

### PR DESCRIPTION
**Pull request by g-stamatis**
Related ticket: https://taskman.eionet.europa.eu/issues/257841

Docusaurus:  https://ci.eionet.europa.eu/job/volto-addons/job/volto-eea-design-system/job/PR-401/2/volto-eea-design-system/
Storybook: https://62713c57bcbdf4004a3d99b8-cfstrojeye.chromatic.com/?path=/story/components-accordion--default

### CODE REVIEW

**Comment by g-stamatis** [comment]
@eea-jenkins build all

File: theme/themes/eea/globals/site.variables [hunk 1]
```diff
@@ -124,7 +124,7 @@
 --------------------*/
 
 /* Global focus-visible for accessibility */
-@focusVisibleOutline: .2rem auto var(--focus-visible, @blue-3);
+@focusVisibleOutline: .2rem auto var(--focus-visible, @blue-2);
```
Comment (ichim-david) [inline theme/themes/eea/globals/site.variables:127]: @g-stamatis why did you change the focus outline to blue-2 from 3? we reference blue 3 for focus in several files when we style the outline

**Review by ichim-david — CHANGES_REQUESTED** [review]
@g-stamatis I am fine with the changes as requested for the ticket but I don't think you should change the focus outline color

File: theme/themes/eea/globals/site.variables (hunk 1 above)
Comment (g-stamatis) [reply theme/themes/eea/globals/site.variables:127]: @ichim-david it was a UX decision to avoid having the same colour between WISE F and the focus outline.

Comment has been added to the ticket now.

Comment (ichim-david) [reply theme/themes/eea/globals/site.variables:127]: @g-stamatis then we also need to make changes in other packages where we've used @blue-3 for outline such as volto-searchlib so yes it's a must for us to get any changes in outline change as that will need to be then tested and changed everywhere as well

Comment (g-stamatis) [reply theme/themes/eea/globals/site.variables:127]: @ichim-david please add your comments at the ticket, where the UX team can see them.

**Review by ichim-david — APPROVED** [review]
